Cargo.toml

```diff
@@ -14,21 +14,28 @@ readme = "README.md"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["opengl", "default_fonts"]
-opengl = ["egui_glow", "baseview/opengl"]
+default = ["wgpu", "default_fonts"]
```

Comment:
I would rather keep the opengl backend as the default since OpenGL is more stable and widely supported than wgpu.

Reply:
i don't 100% agree, because wgpu does also use glow and if it's working correctly it should fall back on it. granted, there was the bug where it just refuses to select an adapter, so i'm not sure.

Reply:
Still, we haven't fully tested the wgpu backend on all platforms so I'd still rather keep opengl as the default for now.

Reply:
that's totally fair